examples/using-MDX/src/pages/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { Link } from "gatsby"
+
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
+
+const IndexPage = () => (
+  <Layout>
+    <SEO title="Home" />
```

Comment:
If you do end up removing the SEO component, the references in this file can be removed too.